packages/components/src/KaizenProvider/KaizenProvider.tsx

```diff
@@ -12,16 +12,30 @@ export type KaizenProviderProps = {
 export const KaizenProvider = ({
   children,
   locale = "en",
-}: KaizenProviderProps): JSX.Element => (
-  <OptionalIntlProvider locale={locale}>
-    <>
-      <ToastNotificationProvider>
-        <ToastNotificationsList />
-        {children}
-      </ToastNotificationProvider>
-      <FontDefinitions />
-    </>
-  </OptionalIntlProvider>
-)
+}: KaizenProviderProps): JSX.Element => {
+  const [documentIsAvailable, setDocumentIsAvailable] = useState<boolean>(false)
+  const [notificationsList, setNotificationsList] = useState<JSX.Element>()
+
+  useEffect(() => {
+    // SSR does not have a document, which is required for ToastNotificationsList.
+    // Await document render before rendering the component.
+    if (document !== undefined) {
+      setNotificationsList(<ToastNotificationsList />)
+      setDocumentIsAvailable(true)
+    }
+  }, [documentIsAvailable])
```

Comment:
Notes:
- If we do not define a dependency array, then we will have an endless amount of renders, which we do not want
- If we define it as `[]`, then the first render will be a step behind and we will not see any notifications which are meant to show on page load